src/PythonMigrationViewExtension/Controls/VisualDifferenceViewer.xaml.cs

```diff
@@ -0,0 +1,57 @@
+﻿using DiffPlex.Wpf.Controls;
+using System.Windows;
+
+namespace Dynamo.PythonMigration.Controls
+{
+    /// <summary>
+    /// Interaction logic for VisualDifferenceViewer.xaml
+    /// </summary>
+    public partial class VisualDifferenceViewer : Window
+    {
+        private PythonMigrationAssistantViewModel ViewModel { get; set; }
+        internal VisualDifferenceViewer(PythonMigrationAssistantViewModel viewModel, Window ownerWindow)
+        {
+            ViewModel = viewModel;
+            Owner = ownerWindow;
+            InitializeComponent();
+            LoadData();
+            DiffView.ViewModeChanged += OnViewModeChanged;
+            DiffView.Loaded += OnDiffViewLoaded;
+        }
+
+        private void OnDiffViewLoaded(object sender, RoutedEventArgs e)
+        {
+            SizeWindowToContent();
+        }
+
+        private void OnViewModeChanged(object sender, DiffViewer.ViewModeChangedEventArgs e)
+        {
+            SizeWindowToContent();
+        }
+
+        private void LoadData()
+        {
+            DiffView.OldText = ViewModel.OldCode;
+            DiffView.NewText = ViewModel.NewCode;
+        }
+
+        private void DiffButton_Click(object sender, RoutedEventArgs e)
+        {
+            if (DiffView.IsInlineViewMode)
+            {
+                DiffView.ShowSideBySide();
+                return;
+            }
+
+            DiffView.ShowInline();
+        }
+
+        private void SizeWindowToContent()
```

Comment:
let's add a comment/xml on why this is needed - would've assumed we have a min width/height and we size the content to the window ?